Domains/4-Application/Connectors/ConceptStation/CSPhysical.ecschema.xml

```diff
@@ -15,75 +15,53 @@
             <SupportedUse>Production</SupportedUse>
         </ProductionStatus>
     </ECCustomAttributes>
-
-    <ECEntityClass typeName="SegmentAssembly" modifier="Abstract" displayLabel="Segment Assembly" description="Base class for groups of components, such as road segments, transition segments, intersections, etc.">
+    <ECEntityClass typeName="ConceptStationPhysicalElement" displayLabel="ConceptStation Physical Element" description="Base class for ConceptStation elements.">
```

Comment:
This class needs to be abstract. It is currently failing the validation check because of it.

Reply:
I agree. I will fix it.